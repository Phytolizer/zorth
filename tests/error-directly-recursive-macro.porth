macro direct
  direct
end

direct
