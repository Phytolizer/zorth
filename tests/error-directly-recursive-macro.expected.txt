:i returncode 1
:i argc 0
:b stdin 0

:b stdout 0

:b stderr 93
tests/error-directly-recursive-macro.porth:2:3: ERROR: macro exceeded expansion limit (1000)

